galaxyproject/galaxy
# Vueify _fieldDialog

#12882 
Converting Dialog field to Vue with ts. 

## How to test the changes?
(Select all options that apply)
  1. Ensure your history has a tabular file
  2. Go to Visualizations from the masthead and select the one called Example
  3. In the right corner, click the double arrows (<<) to open the frame.
  4. Clicking on the open file folder will open the data dialog
  5. Selecting an option will close it. (as will canceling, or clicking out of the modal)
  6. To test multiple, change the value of multiple in `config/plugins/visualizations/example/config/example.xml`, follow the same steps and click OK to see your selected files in the displayed input.
  

## License
